components/identity-core/org.wso2.carbon.identity.base/src/main/java/org/wso2/carbon/identity/base/IdentityBaseUtil.java

```diff
@@ -89,6 +89,9 @@
             "            </wsp:ExactlyOne>" +
             "        </wsp:Policy>";
 
+    private IdentityBaseUtil() {
+    }
+
```

Comment:
unnecessary new line, please remove, we should only have 1 new line separation between two methods.